src/main/java/com/stripe/model/Event.java

```diff
@@ -363,7 +364,7 @@ public static class Data extends StripeObject implements StripeActiveObject {
      */
     @Deprecated
     public StripeObject getObject() {
-      return StripeObject.deserializeStripeObject(object, this.responseGetter);
+      return StripeObject.deserializeStripeObject(object, this.responseGetter, ApiMode.V1);
```

Comment:
Where is the v2 counterpart to this?

Reply:
V2 events don't have a getObject method that deserializes event into an object class. The equivalent would be `fetchRelatedObject` which is already typed.